### PR TITLE
RZR-12: created dockerfile, refactored paths, fixed fonts names

### PR DESCRIPTION
**Pull request by cashidze**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+Out
```
Comment (SashaGraves) [inline .gitignore]: what is it for?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
 # rzrsite-ui
 UI part of the site of our friend
+
+
```
Comment (SashaGraves) [inline README.md:4]: two empty lines

File: .gitignore (hunk 1 above)
Comment (cashidze) [reply .gitignore]: Deprecated. Will be removed

File: README.md (hunk 2 above)
Comment (cashidze) [reply README.md:4]: It's a readme file. Added intentionally to separate our redeyes section from useful sections

**Review by SashaGraves — APPROVED** [review]

**Pull request merged**